src/app/exam/exam-problems/exam-problems.component.ts

```diff
@@ -62,6 +62,6 @@ export class ExamProblemsComponent implements OnInit, AfterViewInit {
   }
 
   public routeToProblem(problemId: number) {
```

Comment:
rename to `routeToQuestion(question: Question)` and pass a question into this function. 

we call it `question` instead of `problem` in the exam context

Reply:
However this function route to `.../problems/...`. Is this fine?